packages/manager/apps/pci-public-ip/src/pages/order/hooks/countries.ts

```diff
@@ -0,0 +1,80 @@
+import { useEffect, useState } from 'react';
+import get from 'lodash.get';
+import { REGIONS } from '@ovh-ux/manager-pci-ssh-keys-app/src/constants';
+import { useRegions } from '@/api/hooks/useRegions';
+import { useProducts } from './useProducts';
+import { useMe } from '@/pages/order/hooks/useMe';
+
+export type TCountry = {
+  name: string;
+  planCode: string;
+  regionNames: string[];
+};
+
+export const useCountries = (projectId: string, regionName: string) => {
+  const [countries, setCountries] = useState<TCountry[]>([]);
+  const { me } = useMe();
+
+  const { isPending: isRegionsPending, data: regions } = useRegions(projectId);
+  const { isPending: isProductspending, data: products } = useProducts();
+
+  useEffect(() => {
+    if (regions && products) {
+      const $products = products.plans.filter(
```

Comment:
$var syntax comes from angularjs, could you remove the "$" from var names ?

Reply:
nope, jQuery popularized this first, i use it to differentiate scope variables from upper scope variables keeping a meaningfull name in the same time, it's not because it's used by jQuery or angularjs that we shouldn't use it